.github/release.yml

```diff
@@ -0,0 +1,33 @@
+changelog:
+  exclude:
+    labels:
+      - bot
+    authors:
+      - renovate
+      - lawnchair-bot
+      - crowdin-bot
+  categories:
+    - title: 🪄 Features
+      labels:
+        - '*'
+      exclude:
+        labels:
+          - dependencies
+          - github_actions
+          - housekeeping
+    - title: 🧹 Housekeeping
+      labels:
+        - housekeeping
+        - github_actions
+    - title: 🧑‍💻 Dependencies
+      labels:
+        - dependencies
+    - title: 🗿 Unclassified
+      labels:
+        - '*'
+      exclude:
+        labels:
+          - enhancement
+          - dependencies
+          - github_actions
+          - housekeeping
```

Comment:
If we can fold them instead of just hiding?

Reply:
Only including the latest entry for each dependency would be ideal.

Reply:
> If we can fold them instead of just hiding?

does it matter